v1/packages/medusa-plugin-webhooks/src/services/webhook.ts

```
@@ -0,0 +1,391 @@
+    const order = await this.orderService_.retrieveWithTotals(orderId, {
+      relations: defaultAdminOrdersRelations,
+      select: defaultAdminOrdersFields,
+    });
+
+    const removedBlacklistedFields = Object.keys(order).reduce((acc, key) => {
+      if (blacklistedRelations.includes(key)) {
+        return acc;
+      }
+
+      return {
+        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)